### PR TITLE
Added 16-17 and 17-18 Committee and Added Social Profile Links

### PR DESCRIPTION
**Pull request by RusherRG**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1788,6 +1852,8 @@ <h1>
   <div class="footer-author">
     <p id="didYouSee" class="author">Did you see our <a href="#" onclick="launchJSTerminal(); return false;">Terminal?</a><span class="icon icon-aerolab"></span></p>
     <p class="author">Crafted by <a href="http://www.kjscecodecell.com" onclick="alert('To understand recursion, one must first understand recursion.')">KJSCE Codecell</a><span class="icon icon-aerolab"></span></p>
+    <small style="color: white; margin-bottom: 0px;">Social Media</small>
+    <p class="author"><a href="https://www.facebook.com/kjscecodecell">Facbook</a><span class="icon icon-aerolab"></span><a href="https://twitter.com/kjsce_codecell">Twitter</a><span class="icon icon-aerolab"></span><a href="https://www.instagram.com/kjsce_codecell">Instagram</a><span class="icon icon-aerolab"></span></p>
```
Comment (nurdtechie98) [inline index.html]: typo facbook->facebook

File: index.html [hunk 2]
```diff
@@ -1764,6 +1764,70 @@ <h1>
...
+        <li><h4>Mr. Ankit Tripathi</h4><p>Technical Team</p></li>
+        <li><h4>Mr. Siddharth Mishra</h4><p>Technical Team</p></li>
+        <li><h4>Mr. Chirag Shetty</h4><p>Technical Team</p></li>
+        <li><h4>Mr. Nishchith Shetty</h4><p>Technical Team</p></li>
+        <li><h4>Mr. Jay Kanakiya</h4><p>Creative Team</p></li>
+        <li><h4>Mr. Raj Shah</h4><p>Creative Team</p></li>
+        <li><h4>Mr. Neel Shah</h4><p>Co-ordinator</p></li>
+        <li><h4>Mr. Shivam Pawase</h4><p>Co-ordinator</p></li>
+        <li><h4>Mr. Karan Agrawal</h4><p>Co-ordinator</p></li>
+        <li><h4>Mr. Nisarg Chandan</h4><p>PR Team</p></li>
```
Comment (nurdtechie98) [inline index.html]: PR team ->PRO

File: index.html [hunk 3]
```diff
@@ -1764,6 +1764,70 @@ <h1>
...
+        <li><h4>Ms. Brinda Ashar</h4><p>PR Team</p></li>
```
Comment (nurdtechie98) [inline index.html]: PR team -> Joint PRO

**Comment by nurdtechie98** [comment]
@RusherRG sorry for the PR team changes rest seems good to merge

**Comment by inishchith** [comment]
closes #66 . @nurdtechie98 approve !

**Pull request merged**